package.json

```diff
@@ -38,6 +84,7 @@
     "node-sass": "^4.12.0",
     "npm-run-all": "^4.1.5",
     "rimraf": "^3.0.0",
-    "sassdoc": "^2.7.1"
+    "sassdoc": "^2.7.1",
+    "standard-version": "^7.0.0"
```

Comment:
Avoid using `^` and `~` in `package.json` and use fixed versions instead.
Despite `^` being the default option, set it to fixed versions with `.npmrc` and `save-exact=true`

```
// .npmrc
save-exact=true
```

Not part of this PR, but consider creating a separate PR to do it. In order to update, use a tool like [ncu](https://www.npmjs.com/package/npm-check-updates)